tests/PSPIntegrationTests.js

```diff
@@ -17,6 +17,9 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
 var fluid = require("infusion"),
     gpii = fluid.registerNamespace("gpii");
 
+// TODO: Don't commit this.
```

Comment:
These two debugging lines can be removed.

Reply:
Okay, but I didn't put them there :-).  I think my merge of the GPII-3619 pre-dates its merge into master, where this TODO was removed.

Reply:
Yes, this one was my fault and has already been removed in later work.